smsjwplatform/urls.py

```diff
@@ -2,18 +2,15 @@
 Default URL patterns for the :py:mod:`smsjwplatform` application are provided by the
 :py:mod:`.urls` module. You can use the default mapping by adding the following to your global
 ``urlpatterns``:
-
 .. code::
-
```

Comment:
This blank line is required by the RST specification to separate options from content.